base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2736,10 +2736,12 @@ impl UniqueIdIndexKey {
     /// `parent_public_key` - the parent asset public key to which the token is assigned
     /// `unique_id` - a series of bytes representing the token uniquely for the asset
     pub fn new(parent_public_key: Option<&PublicKey>, unique_id: &[u8]) -> Self {
-        let unique_id_hash = Blake256::default().chain(unique_id).finalize();
+        let unique_id_hash = base_layer_core_chain_storage_lmdb_hasher::<Blake256>(LMDB_STORAGE_HASH_LABEL)
```

Comment:
As per simplification:
```suggestion
        let unique_id_hash = CoreChainStorageHasherBlake256::new()
```